### ISSUE TITLE
Apocalypse fixes (again)

### ISSUE THREAD

**Issue by Tarr-OHOL** [jasonrohrer/OneLifeData7#1243]
Again, we have people trying to do the impossible to stop apocalypses which of course is annoying for others to try to deal with. 

1). Population restriction did not stop this. Why? 6 people or whatever have a pool of 70+ accounts to artificially reach the limits required to start (but not finish) the apocalypse. Maybe every stage needs this and not just the first? This isn't going to fix the issue due to the amount of accounts vs actual players online generally. Again, this will not solve the issue and more trying to get the idea of what happened this last time.

2). This would have been unstoppable if the setting wasn't randomly off due to time. If the fight for the apocalypse is driving a ghost 12+ hours away to stop the timer for the apocalypse, why do we even have a required time limit? I don't think anyone actively wants to be trying to spam trucker babies or roll rng for a ghost. The apocalypse does give a time warning after 12 hours but what do you do when you need those 12 hours to drive? I would say lower the time but then this will just get spammed at nonpeak hours. 

3). Donkeys, and flight. Basically donkeys were birthing other donkey accounts which leads us back to point one. Using a plane in combination with a ghost to get where they were going no one would be able to stop this. Removing unlimited flight again would at least disable ghosts from this strat.

I could list more obviously but lets just take a small peak at our issues:

Population
Time
Area

We can start with time since I think both sides of the coin can agree.

No one is going to find 12+ hours of driving a truck or sports car as fun or engaging gameplay. The drive was done by someone essentially holding down mouse or direction keys while sleeping for 50 minutes at a time then setting an alarm to make sure not to run out of gas or die. There is no way this is an intended way to do something in game and frankly it's kind of a ridiculous way to do things. I would say shorten the apocalypse required time but then the current meta to try to force it is going to be either driving further or still driving even further to once again stop people from being able to make it. 

Population is obviously not a deterrent because with as many accounts as the "bad" guys have means they can artificially inflate it to pretty much any required number. With 70 accounts even requiring have the population for apocalypse means we need over 70 regular accounts just to reach a 50% threshold away from their project. I don't even know whether there should be a population restriction for this case because there are too many "bad" accounts to really stop anyways. Racial restriction for each block? Maybe to stab the apocalypse with a bloody sword? That would at least require elders to start a way but that's still so easy. 

Area is supposed to be an already fixed issue which was fixed through population requirements except you can still start the apocalypse in the middle of nowhere with the extra accounts. Obviously no one is banned for being an asshat so instead they'll just buy accounts. This is fine too, support the creator why don't you. But of course this is bypassed with a combination of tons of accounts, a ghost in a truck, and an airplane. I think in this case flight just has to be limited again. You would still be able to set up multiple pads every X tiles for a bunch of small flights but at least this means they need to stop.

I hate to bring the same old issue up but there has to be a way to stop AND do the apocalypse. This last one only got foiled because the apocalypse was turned off on BS2. The apocalypse should always be doable given when needed (and it is). It's not fair for either party given that the apocalypse group did (cheat) an apocalypse through and should have gotten their apocalypse but at the same time people need to be able to stop is if they want to. 

TL;DR The current issue is a combination of ghosts driving too far, donkeys flying donkey and their alt accounts to an area too far to do anything against. I am sure both sides can agree that this isn't fun for either party.

**Comment by Tarr-OHOL** [comment]
So revisiting this issue after the apocalypse tonight. 

Time: What if you could only remove one tier per six hours just like you can only add one per six hours? This seems more fair for each side of the situation as defenders of the apocalypse know they're safe to leave it once a tier is removed and attackers can't just completely nullify the apocalypse crews efforts. 

Population: I don't honestly think this should be a thing due to the amount of accounts running around between a small group of players. We're at the point that unless there's a steam sale they can beat the threshold artificially by bringing donkey accounts to an area and just sit around. If population stays donkey accounts shouldn't count because changes were made to stop donkeys from doing apocalypses in the first place (along with the bells.)

Area: This is actually probably the simplest. Apocalypse shouldn't be able to go off outside of 10k or some other smallish distance like how it was done to stop donkeys from doing it. This current iteration of the apocalypse is by far the worst. There's no way driving 12 irl hours east on the map as a ghost should be viable.

**Comment by Forman9** [comment]
![image](https://github.com/user-attachments/assets/2b58cd9a-ff81-49b7-af3f-a2b9350f9098)
